src/aleph/web/controllers/version.py

```diff
@@ -0,0 +1,9 @@
+from aiohttp import web
+from aleph import __version__
```

Comment:
We should decide if the main way to get the version is through `aleph.__version__` or `aleph.get_git_version()`. If we go for the first option, I would rename `get_git_version` -> `_get_git_version` to mark it as private.

Reply:
Works for me, __version__ is more Pythonic IMO.